# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback definition
bool found
bool foundSure
int32 tag_x
int32 tag_y

